### ISSUE TITLE
loss_rank

### ISSUE THREAD

**Issue by Heyuting-Creator** [ZHU-Zhiyu/High-Rank_RGB-Event_Tracker#5]
Hello, thank you very much for your open source work. I have a little question: about " def loss_rank(self, outputs, targetsi, temp_annoi=None).
        """Compute the losses related to the bounding boxes, the L1 regression loss and the GIoU loss
           The target dicts must contain the key "boxes" containing a tensor of dim [nb_target_boxes, 4].
           The target boxes are expected in format (center_x, center_y, h, w), normalized by the image size.
        """
        attn = outputs['attn']
        # print('attn shape:{}'.format(attn.shape))
        attn1 = torch.cat([attn[:,:,114:344,57:114], attn[:,:,114:344,344:]],dim=3)
        attn1 = attn1.mean(dim=0, keepdim=True).mean(dim=1, keepdim=True)
        attn2 = torch.cat([attn[:,:,344:,:57], attn[:,:,344:,114:344]],dim=3)
        attn2 = attn2.mean(dim=0, keepdim=True).mean(dim=1, keepdim=True)", why are attn1 and attn2 taken this way?

**Comment by ZHU-Zhiyu** [comment]
Hi @Heyuting-Creator, you can refer to Fig. 7 in our paper. The green regions in the attention matrix correspond to the cross-model interaction. Thus, we only regularize those parts.
![image](https://github.com/ZHU-Zhiyu/High-Rank_RGB-Event_Tracker/assets/31000629/792c29a6-72d8-49a8-adda-1722dc634a2d)

**Comment by Heyuting-Creator** [comment]
> Hi @Heyuting-Creator, you can refer to Fig. 7 in our paper. The green regions in the attention matrix correspond to the cross-model interaction. Thus, we only regularize those parts. ![image](https://private-user-images.githubusercontent.com/31000629/333462836-792c29a6-72d8-49a8-adda-1722dc634a2d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY1NDk4NjcsIm5iZiI6MTcxNjU0OTU2NywicGF0aCI6Ii8zMTAwMDYyOS8zMzM0NjI4MzYtNzkyYzI5YTYtNzJkOC00OWE4LWFkZGEtMTcyMmRjNjM0YTJkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI0VDExMTkyN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWExMDM2MzE3OGI1NGM3MDFiN2Q1ZDliMjQ4MjM4MGE0MWQyMmQ1N2MwNTQwMzZkOWMxMWY0ZjkwYTdlZWUxNTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.E8dEVnIl6p2BwFI-49dhRpcI95ddCfhHhDmFcTOX4K0)

Got it, thank you very much for your reply.
